### PR TITLE
[Base] Units and Uui: remove unused includes

### PR DESCRIPTION
**Pull request by donovaly**

### CODE REVIEW

File: src/Base/UnitsSchemaImperial1.cpp [hunk 1]
```diff
@@ -30,12 +30,8 @@
 #endif
 
 #include <QString>
-#include "Console.h"
-#include "Exception.h"
-#include "UnitsApi.h"
+
 #include "UnitsSchemaImperial1.h"
-#include <cmath>
```
Comment (chennes) [inline src/Base/UnitsSchemaImperial1.cpp:37]: This fails to compile on Linux, it needs the math functions `round` and `floor`.

Comment (donovaly) [reply src/Base/UnitsSchemaImperial1.cpp:37]: Thanks. I re-added it now

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6441">pipeline status</a> for feature branch [PR_6441](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6441). Pipeline [478298457](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/478298457) was triggered at [971ae57](https://github.com/FreeCAD/FreeCAD/pull/6441/commits/971ae57202bc81dc4c6c073763aa67e079cb2372). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by wwmayer** [comment]
9ec0bcdcf2

**Pull request closed without merging**